snuba/datasets/factory.py

```diff
@@ -37,28 +42,30 @@ def get_dataset(name: str) -> Dataset:
             f"dataset {name!r} is not available in this environment"
         )
 
-    from snuba.datasets.events import EventsDataset
-    from snuba.datasets.errors import ErrorsDataset
     from snuba.datasets.cdc.groupassignee import GroupAssigneeDataset
     from snuba.datasets.cdc.groupedmessage import GroupedMessageDataset
-    from snuba.datasets.transactions import TransactionsDataset
+    from snuba.datasets.discover import DiscoverDataset
+    from snuba.datasets.events import EventsDataset
+    from snuba.datasets.errors import ErrorsDataset
+    from snuba.datasets.groups import Groups
+    from snuba.datasets.querylog import QuerylogDataset
     from snuba.datasets.outcomes import OutcomesDataset
     from snuba.datasets.outcomes_raw import OutcomesRawDataset
-    from snuba.datasets.groups import Groups
-    from snuba.datasets.discover import DiscoverDataset
     from snuba.datasets.sessions import SessionsDataset
+    from snuba.datasets.transactions import TransactionsDataset
```

Comment:
If you are fixing the order, could you fix all of them ?

Reply:
Sure